data/entities.js

```diff
@@ -12048,4 +12048,19 @@ module.exports = [
     categories: ['consent-provider'],
     domains: ['webads.mgr.consensu.org', '*.webads.mgr.consensu.org'],
   },
+  {
+    name: 'Trustcommander',
+    company: 'Commandersact',
+    homepage: 'https://www.commandersact.com',
+    categories: ['consent-provider'],
+    domains: ['*.trustcommander.net'],
+    examples: ['cdn.trustcommander.net', 'privacy.trustcommander.net'],
+  },
+  {
+    name: 'Hubvisor',
+    homepage: 'https://www.hubvisor.io',
+    categories: ['ad'],
+    domains: ['*.hubvisor.io'],
+    examples: ['stream.hubvisor.io', 'stream.hubvisor.io'],
```

Comment:
were these meant to be different examples?

Reply:
sorry it's a mistaken i would write :  ['cdn.hubvisor.io', 'stream.hubvisor.io']

i can do another PR

Reply:
no need for a new PR :) thanks for the clarification
```suggestion
    examples: ['cdn.hubvisor.io', 'stream.hubvisor.io'],
```